templates/nextjs/package.json

```diff
@@ -31,6 +31,7 @@
     "eslint-config-next": "14.2.4",
     "postcss": "^8",
     "tailwindcss": "^3.4.4",
-    "typescript": "^5"
+    "typescript": "^5",
+    "vitest": "^1.6.0"
```

Comment:
Should the template's `package.json` have a pre-configured `test` script that users can just `pnpm test`? 🙂

Reply:
Good catch... I tested this via `pnpm test:filter`

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2755/commits/eec4be4f3bbf2b3e34034b5378dd21a55fd1721c